core/src/main/resources/error/error-classes.json

```diff
@@ -1651,6 +1665,11 @@
     ],
     "sqlState" : "46110"
   },
+  "NOT_SUPPORTED_COMMAND_WITHOUT_HIVE_SUPPORT" : {
+    "message" : [
+      "<cmd> is not supported, if you want to enable it, please set `spark.sql.catalogImplementation` to `hive`."
```

Comment:
Please, quote the SQL config and its value in the same way as `toSQLConf` and `toSQLConfVal`